scripts/linters/pylint_extensions.py

```diff
@@ -1484,4 +1484,4 @@ def register(linter):
     linter.register_checker(DivisionOperatorChecker(linter))
     linter.register_checker(SingleLineCommentChecker(linter))
     linter.register_checker(DocstringChecker(linter))
-    linter.register_checker(BlankLineBelowFileOverviewChecker(linter))
+    # linter.register_checker(BlankLineBelowFileOverviewChecker(linter))
```

Comment:
Why is this commented?

Reply:
This was commented as an escape from errors fixed in [#8792](https://github.com/oppia/oppia/pull/8792)